xmppserver/src/main/java/org/jivesoftware/openfire/muc/MUCEventDispatcher.java

```diff
@@ -122,6 +122,16 @@ public static void roomDestroyed(JID roomJID) {
         }
     }
 
+    public static void roomClearChatHistory(JID roomJID) {
+        for (MUCEventListener listener : listeners) {
+            try {
+                listener.roomClearChatHistory(roomJID);
+            } catch (Exception e) {
+                Log.warn("An exception occurred while dispatching a 'roomClearChatHistory' event!", e);
```

Comment:
Please add the room JID to the logged message, which is going to be very helpful when we get to analyze log files that contain this warning.

Reply:
Done.